cloud-sql/postgres/sqlalchemy/main.py

```diff
@@ -27,34 +27,12 @@
 db_name = os.environ.get("DB_NAME")
 cloud_sql_connection_name = os.environ.get("CLOUD_SQL_CONNECTION_NAME")
 
-app = Flask(__name__)
-
-logger = logging.getLogger()
-
-# [START cloud_sql_postgres_sqlalchemy_create]
-# The SQLAlchemy engine will help manage interactions, including automatically
-# managing a pool of connections to your database
-db = sqlalchemy.create_engine(
-    # Equivalent URL:
-    # postgres+pg8000://<db_user>:<db_pass>@/<db_name>?unix_sock=/cloudsql/<cloud_sql_instance_name>/.s.PGSQL.5432
-    sqlalchemy.engine.url.URL(
-        drivername='postgres+pg8000',
-        username=db_user,
-        password=db_pass,
-        database=db_name,
-        query={
-            'unix_sock': '/cloudsql/{}/.s.PGSQL.5432'.format(
-                cloud_sql_connection_name)
-        }
-    ),
-    # ... Specify additional properties here.
-    # [START_EXCLUDE]
-
+db_config = {
```

Comment:
This config dict should probably be moved to right above where we create the connection pool. (same goes for the MySQL sample)
I suggest initializing it inside `init_connection_engine` instead of making it a global variable, and then passing it into the `init_tcp` and `init_unix_socket` functions as a parameter